configs/typescript.yaml

```
@@ -19,3 +19,4 @@ rules:
   '@typescript-eslint/no-extra-semi': 0
   no-extra-parens: 0
   no-use-before-define: 0
+  '@typescript-eslint/no-unused-vars': 2
```

I'd keep them sorted:
```suggestion
  '@typescript-eslint/no-extra-semi': 0
  '@typescript-eslint/no-unused-vars': 2
  no-extra-parens: 0
  no-use-before-define: 0
```